common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -547,8 +547,6 @@ public static String ipToIPv6Str(byte[] ip) {
                             if (ipV6Check(ipByte)) {
                                 if (!isInternalV6IP(ip)) {
                                     return ipByte;
-                                } else if (internalIP == null) {
```

Comment:
Is it possible that there is only internal IPv6 address? If it exists, the IP address cannot be obtained.

Reply:
yep,  what I thought was that if no address was selected here, use fakeIp instead.